agent-core/src/main/java/com/newrelic/agent/android/analytics/EventManagerImpl.java

```diff
@@ -363,7 +385,7 @@ public void onEventQueueSizeExceeded(int currentQueueSize) {
     /**
      * Notified when event buffer flush time has been exceeded. This normally occurs
      * between harvest cycles.
-     *
+     * <p>
```

Comment:
Code cleanup replaces this in JavaDoc comments. Should be removed, but doesn't do any harm.